spec/workers/subscription_content_worker_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe SubscriptionContentWorker do
```

Comment:
There is no test which calls the Sidekiq `perform_async` method which I suspect will fail as the worker does not `include Sidekiq::Symbols`.

Reply:
I think `.perform_async` is in `Sidekiq::Worker` isn't it?

Reply:
Oh... It's needed for keyword arguments. 🙊

Reply:
okedokey - try that